classes/FaviconController.php

```diff
@@ -13,6 +12,13 @@ public function index()
     {
         $settings = Settings::instance();
 
+        if (!$settings->webmanifest_enabled) {
```

Comment:
combine these two if's into one, something like:
`if (!$settings->webmanifest_enabled || !$settings->favicon_enabled)`

Reply:
This operator won't logically fit there.